src/handlers/cell.ts

```diff
@@ -144,6 +147,27 @@ export class CellManager implements IDisposable {
 
     editor.editor.on('gutterClick', this.onGutterClick);
     editor.editor.on('renderLine', this.onNewRenderLine);
+
+    this.setHover();
+  }
+
+  setHover() {
```

Comment:
Is it impossible to do this work with pure CSS? Doing it with JS seems like overkill.